### ISSUE TITLE
Feature Request: Add site address in the app log for login flow

### ISSUE THREAD

**Issue by reginabally** [wordpress-mobile/WordPress-iOS#16751]
Currently, when a user was trying to login into the app using the "Enter your existing site address" flow, the site address will not be recorded in the app log.

Sample log output for a login event with a site address in the iOS app:

```
2021-06-22 13:25:40:849 🔵 Tracked: unified_login_interaction <click: login_with_site_address, flow: prologue, source: default, step: prologue>
2021-06-22 13:25:41:422 🔵 Tracked: unified_login_step <flow: login_site_address, source: default, step: start>
2021-06-22 13:25:49:166 TracksService sendQueuedEvents completed. Sent 7 events.
2021-06-22 13:25:55:913 🔵 Tracked: unified_login_interaction <click: submit, flow: login_site_address, source: default, step: start>
2021-06-22 13:25:55:944 Assume the given url is the home page and XML-RPC sits at /xmlrpc.php
2021-06-22 13:26:03:977 🔵 Tracked: unified_login_step <flow: login_site_address, source: default, step: username_password>
2021-06-22 13:26:04:470 TracksService sendQueuedEvents completed. Sent 2 events.
2021-06-22 13:26:06:535 <WordPress.WordPressAppDelegate: 0x282284480> applicationWillResignActive(_:)
2021-06-22 13:26:07:822 <WordPress.WordPressAppDelegate: 0x282284480> applicationDidBecomeActive(_:)
2021-06-22 13:26:54:141 🔵 Tracked: unified_login_interaction <click: submit, flow: login_site_address, source: default, step: username_password>
2021-06-22 13:26:54:175 Assume the given url is the home page and XML-RPC sits at /xmlrpc.php
2021-06-22 13:26:57:042 🔵 Tracked: login_failed_to_login <error_code: 0, error_description: Error Domain=WordPressKit.WordPressOrgXMLRPCApiError Code=0 "(null)" UserInfo={WordPressOrgXMLRPCApiErrorKeyData={length = 410, bytes = 0x3c3f786d 6c207665 7273696f 6e3d2231 ... 73706f6e 73653e0a }, WordPressOrgXMLRPCApiErrorKeyStatusCode=403, WordPressOrgXMLRPCApiErrorKeyDataString=<?xml version="1.0" encoding="UTF-8"?>
<methodResponse>
  <fault>
    <value>
      <struct>
       }, error_domain: WordPressKit.WordPressOrgXMLRPCApiError>
2021-06-22 13:26:57:065 The operation couldn’t be completed. (WordPressKit.WordPressOrgXMLRPCApiError error 0.)
```

With the site address logged in the app log, it will save time in identifying the site address and troubleshooting.

**Comment by aerych** [comment]
Should be available in the 17.7 release. Wrangled in 
https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/601 
https://github.com/wordpress-mobile/WordPress-iOS/pull/16747

@reginabally do you know if a similar change is needed for the Android app?

**Comment by reginabally** [comment]
Thanks for the quick fix, @aerych! As for Android, we can see the site addresses in the app log even before the user was successfully logged into the app, so we don't need it for the Android app yet.
